components/tools/OmeroPy/test/integration/metadata/test_populate.py

```diff
@@ -400,7 +400,8 @@ class TestPopulateMetadata(lib.ITest):
     METADATA_IDS = [x.__class__.__name__ for x in METADATA_FIXTURES]
 
     @mark.parametrize("fixture", METADATA_FIXTURES, ids=METADATA_IDS)
-    def testPopulateMetadata(self, fixture):
+    @mark.parametrize("batch_size", (None, 10, 1000))
```

Comment:
RFE for sometime in the future: From past experience writing this sort of thing in omero-features it's useful to have your input test data be at least 2*batch size but not exactly divisible.